tests/test_frontdoor.py

```diff
@@ -280,6 +285,12 @@ def _assert_change_button_is_hidden(page, page_name):
     assert change is None
 
 
+def _assert_change_button_is_not_hidden(page, page_name):
+    elements = page.all("a:contains('Change')")
+    change = next(filter(lambda el: f"/{page_name}/change/" in el.attrib["href"], elements), None)
+    assert change is not None
```

Comment:
considering this is now the 3rd time this logic is expressed may be time to extract to a helper